src/DynamicData.Tests/Cache/RightJoinFixture.cs

```diff
@@ -69,15 +69,15 @@ public void AddRightOnly()
             _right.Edit(
                 innerCache =>
                     {
-                        innerCache.AddOrUpdate(new DeviceMetaData("Device1"));
-                        innerCache.AddOrUpdate(new DeviceMetaData("Device2"));
-                        innerCache.AddOrUpdate(new DeviceMetaData("Device3"));
+                        innerCache.AddOrUpdate(new DeviceMetaData(1,"Device1"));
```

Comment:
You will get syntax issues with these, you need a space after the 1, etc.

Reply:
as far as I see not in the test project...there's no StyleCop there
or am I missing something?

Reply:
There should be stylecop throughout the entire application. The test project is just a cut down version, eg doesn't require comments etc.

Reply:
I will do a PR with the editorconfig tomorrow to make it more consistent, so if your stuff gets merged in I'll handle the style fixes.

@RolandPheasant usually likes to review these ones personally.